src/app/pages/review-attributes/_services/review-attributes.model.service.ts

```diff
@@ -1,33 +1,64 @@
-import { Injectable } from "@angular/core";
+import { Injectable } from '@angular/core';
+import {AuthService, JWTApiService} from '@savvato-software/savvato-javascript-services';
 import { ReviewAttributesApiService } from "./review-attributes.api.service";
-import { ModelTransformingService } from "@savvato-software/savvato-javascript-services";
+import { Constants } from "../../../_constants/constants";
 
 @Injectable({
-  providedIn: "root",
+    providedIn: 'root'
 })
 export class ReviewAttributesModelService {
-  model: any = {};
-
-  constructor(
-    private _modelTransformingService: ModelTransformingService,
-    private _reviewAttributesApiService: ReviewAttributesApiService
-  ) {}
-
-  init() {
-    this._modelTransformingService.clearAllTransformers();
-
-    this._modelTransformingService.addTransformer((model, done) => {
-      this._reviewAttributesApiService
-        .getNextReviewEligiblePhrase()
-        .then((response) => {
-          model["nextReviewPhrase"] = response;
-          done();
-        });
-    });
-    this._modelTransformingService.transform(this.model);
-  }
-
-  getNextReviewEligiblePhrase() {
-    return this.model["nextReviewPhrase"];
-  }
+
+    model: any = {};
+    reasonList: any = [];
+    saveReview: any = {};
+
+    constructor(private _apiService: JWTApiService,
+                private _authService: AuthService,
+                private ReviewAttributesApiService: ReviewAttributesApiService,
+                private _constants: Constants) {
+
+    }
+
+
+    getNewPhrase() {
+        return new Promise((resolve, reject) => {
```

Comment:
No need to wrap this in a promise.. you can just return the promise from the api.service.